packages/demand-bloc-payment/src/providers/adyenErrors.ts

```diff
@@ -1,5 +1,13 @@
 import { HttpResponseOk, PaymentAuthResponse } from '@karhoo/demand-api'
 
+export class AdyenProviderError extends Error {
```

Comment:
For the sake of consistency would be good to name it `AdyenError` like [it is named for Braintree](https://github.com/karhoo/web-lib-demand/blob/master/packages/demand-bloc-payment/src/types.ts#L1)